recipes/icechunk/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "icechunk" %}
+{% set version = "0.1.0a10" %}
```

Comment:
This one falls under https://github.com/conda-forge/cfep/blob/main/cfep-05.md

Reply:
unless i'm missing something, this guide doesn't seem to have any examples of how to apply this to packages in `staged-recipes` before they have a feedstock repo. 

any tips on how to do this?